### PR TITLE
Set collaborative attribute to False in Jupyterlite

### PR DESCRIPTION
**Pull request by martinRenou**
This has the side effect of fixing the saving of files in JupyterLite for now

### CODE REVIEW

File: packages/schema/src/model.ts [hunk 1]
```diff
@@ -40,7 +40,8 @@ export class JupyterCadModel implements IJupyterCadModel {
     }
   };
 
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] != undefined;
```
Comment (trungleduc) [inline packages/schema/src/model.ts:44]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```

File: python/jupytercad_core/src/jcadplugin/modelfactory.ts [hunk 2]
```diff
@@ -19,7 +19,8 @@ export class JupyterCadJcadModelFactory
   /**
    * Whether the model is collaborative or not.
    */
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] != undefined;
```
Comment (martinRenou) [inline python/jupytercad_core/src/jcadplugin/modelfactory.ts:23]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```

File: python/jupytercad_core/src/stepplugin/modelfactory.ts [hunk 3]
```diff
@@ -11,7 +11,8 @@ export class JupyterCadStepModelFactory
   /**
    * Whether the model is collaborative or not.
    */
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] != undefined;
```
Comment (martinRenou) [inline python/jupytercad_core/src/stepplugin/modelfactory.ts:15]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```

File: packages/schema/src/model.ts [hunk 4]
```diff
@@ -40,7 +40,8 @@ export class JupyterCadModel implements IJupyterCadModel {
     }
   };
 
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```
Comment (martinRenou) [inline packages/schema/src/model.ts:44]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] === undefined;
```

File: python/jupytercad_core/src/jcadplugin/modelfactory.ts [hunk 5]
```diff
@@ -19,7 +19,8 @@ export class JupyterCadJcadModelFactory
   /**
    * Whether the model is collaborative or not.
    */
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```
Comment (martinRenou) [inline python/jupytercad_core/src/jcadplugin/modelfactory.ts:23]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] === undefined;
```

File: python/jupytercad_core/src/stepplugin/modelfactory.ts [hunk 6]
```diff
@@ -11,7 +11,8 @@ export class JupyterCadStepModelFactory
   /**
    * Whether the model is collaborative or not.
    */
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```
Comment (martinRenou) [inline python/jupytercad_core/src/stepplugin/modelfactory.ts:15]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] === undefined;
```

File: python/jupytercad_core/src/stlplugin/modelfactory.ts [hunk 7]
```diff
@@ -11,7 +11,8 @@ export class JupyterCadStlModelFactory
   /**
    * Whether the model is collaborative or not.
    */
-  readonly collaborative = true;
+  readonly collaborative =
+    document.querySelectorAll('[data-jupyter-lite-root]')[0] !== undefined;
```
Comment (martinRenou) [inline python/jupytercad_core/src/stlplugin/modelfactory.ts:15]:
```suggestion
    document.querySelectorAll('[data-jupyter-lite-root]')[0] === undefined;
```

**Comment by martinRenou** [comment]
Working! Let's go ahead with this

**Pull request merged**